test/school_house_web/views/html_helpers_test.exs

```diff
@@ -0,0 +1,57 @@
+defmodule SchoolHouseWeb.HtmlHelpersTest do
+  use SchoolHouseWeb.ConnCase
+
+  alias SchoolHouseWeb.HtmlHelpers
+
+  describe "avatar_url/1" do
+    test "appends .png to GitHub profile link" do
+      assert "https://github.com/elixirschool.png" == HtmlHelpers.avatar_url("https://github.com/elixirschool")
+    end
+  end
+
+  describe "current_locale/0" do
+    setup do
+      Gettext.put_locale("es")
+
+      on_exit(fn ->
+        Gettext.put_locale("en")
+      end)
+    end
```

Comment:
This could potentially live at the top of the file since. We can always rely on the `es` locale in testing since it has great coverage in the lessons.